package.json

```diff
@@ -152,6 +152,11 @@
 				"title": "Login",
 				"category": "Sourcery"
 			},
+			{
+				"command": "sourcery.login.choose",
+				"title": "Login Choose",
```

Comment:
Rename to login, and hide the other command in the command palette (see below in this file for how to do this).

At the moment in the command palette you get both options - I think this one is better since it gives you a choice.